bitcoin/esplora/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:14-alpine3.13 AS build
+
+RUN apk add --update --no-cache \
+    git \
+    bash
+
+WORKDIR /workdir
+
+RUN git clone --no-checkout https://github.com/Blockstream/esplora.git
+
+WORKDIR esplora
+
+RUN git checkout f6da494a9a0080ae2ae13842bcf53b259e3a3360
+
+RUN npm install
+
+ENV CORS_ALLOW="*"
+
+ARG API_URL
+ENV API_URL=$API_URL
```

Comment:
Do these environment variables get encoded into the dist, or does declaring them in the build layer persist to the run layer? If it's the later, I think it makes more sense to have these ARG and ENV commands in the run layer.